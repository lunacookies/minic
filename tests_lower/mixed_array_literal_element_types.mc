func main {
	x := 0
	a := [1, &x, [], 2]
}
