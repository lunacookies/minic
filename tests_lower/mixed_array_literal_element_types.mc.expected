func main
	var x i64
	var a [4]i64
	{
		set x = 0
		set a = [
			1,
			<missing>,
			<missing>,
			2,
		]
	}
tests_lower:30..32: error: expected “i64” but found “*i64”
tests_lower:34..36: error: expected “i64” but found “[0]void”
tests_lower:13..19: warning: unused variable
tests_lower:21..40: warning: unused variable
